### PR TITLE
docs(README.md): update nixos instructions

### PR DESCRIPTION
**Pull request by rubiigen**
found that the nixos instructions proposed don't work on some configs. This should unify it into something thatll work on the vast majority.

### CODE REVIEW

**Comment by bubylou** [comment]
Can confirm this worked for me on NixOS and was greatly appreciated as a new Nix user. FYI you can drop the `pkgs.` from the `pkgs.catpuccin-gtk`.

File: README.md [hunk 1]
```diff
@@ -51,12 +51,18 @@ By default, the variant is `frappe`, the accent is `blue`, the size is `standard
...
-pkgs.catppuccin-gtk.override {
-  accents = [ "pink" ]; # You can specify multiple accents here to output multiple themes
-  size = "compact";
-  tweaks = [ "rimless" "black" ]; # You can also specify multiple tweaks here
-  variant = "macchiato";
-}
+# as per wiki examples
+environment.systemPackages = with pkgs; [
+  ...
+  (pkgs.catppuccin-gtk.override {
```
Comment (rubyowo) [inline README.md:57]:
```suggestion
  (catppuccin-gtk.override {
```

File: README.md [hunk 2]
```diff
@@ -51,12 +51,18 @@ By default, the variant is `frappe`, the accent is `blue`, the size is `standard
 
 Example:
 ```nix
-pkgs.catppuccin-gtk.override {
-  accents = [ "pink" ]; # You can specify multiple accents here to output multiple themes
-  size = "compact";
-  tweaks = [ "rimless" "black" ]; # You can also specify multiple tweaks here
-  variant = "macchiato";
-}
+# as per wiki examples
```
Comment (rubyowo) [inline README.md:54]: Can you add a link to the wiki page this is from?

**Review by rubyowo — CHANGES_REQUESTED** [review]
Hiya! we've just merged another PR that updates the nix instructions in the README, can you rebase?.

File: README.md [hunk 3]
```diff
@@ -51,12 +51,18 @@ By default, the variant is `frappe`, the accent is `blue`, the size is `standard
...
+  ...
+  (pkgs.catppuccin-gtk.override {
+    accents = [ "pink" ]; # You can specify multiple accents here to output multiple themes
+    size = "compact";
+    tweaks = [ "rimless" "black" ]; # You can also specify multiple tweaks here
+    variant = "macchiato";
+  })
+  ...
+];
+# again, this is pulling from a nixos wiki example and is made to work with most nixos configrations
```
Comment (rubyowo) [inline README.md:65]: Unnecessary comment.

**Review by rubyowo — CHANGES_REQUESTED** [review]

File: README.md (hunk 2 above)
Comment (rubiigen) [reply README.md:54]: i used an example from nixos.wiki on vencord with discord overriding. https://nixos.wiki/wiki/Discord

**Comment by rubiigen** [comment]
removed the unnecessary comment, provided a source and dropped pkgs.

**Review by rubyowo — APPROVED** [review]
LGTM

**Pull request merged**